### DISCUSSION TITLE
How to register DSC v6 resources at a Broker?

### DISCUSSION THREAD

**Discussion by sebplorenz** [International-Data-Spaces-Association/DataspaceConnector#84] [Q&A] [answered]
Hi,
I tried to register my DSC version 6 at a Broker. The connector itself gets registered. But its resources are missing.
When looking into the ConnectorUpdateMessage that is sent from the DSC to the Broker, I can see that the id of the resource catalog is submitted. But not the resource descriptions.
What do I have to do to update the resources too?
Regards, Sebastian

**Comment by juliapampus** [comment, marked as answer]
https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/IdsEcosystem#ids-metadata-broker
